src/tools/interop/idt/capture/file_utils.py

```diff
@@ -0,0 +1,58 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+import time
+from pathlib import Path
+from typing import TextIO
+
+
+def warn_if_file_not_growing() -> None:
```

Comment:
I would try to avoid adding in deadcode. It looks like this isn't called anywhere, yet. I like the idea of the function though.

Reply:
Thanks, done!